agents/email_agent.py

```
@@ -0,0 +1,333 @@
+import asyncio
+import json
+import re
+import email
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+# from datetime import datetime
+from typing import List, Dict, Any
+
+from langchain_openai import ChatOpenAI
+from langchain.prompts import PromptTemplate
```

Ruff (0.12.2)

`langchain.prompts.PromptTemplate` imported but unused

Remove unused import: `langchain.prompts.PromptTemplate`

(F401)